specs/merge/beacon-chain.md

```diff
@@ -194,9 +209,9 @@ def process_execution_payload(state: BeaconState, body: BeaconBlockBody) -> None
         assert execution_payload.parent_hash == state.latest_execution_payload_header.block_hash
         assert execution_payload.number == state.latest_execution_payload_header.number + 1
 
-    timestamp = compute_time_at_slot(state, state.slot)
-    execution_state_root = state.latest_execution_payload_header.state_root
-    execution_state_transition(execution_state_root, body.execution_payload, timestamp)
+    assert execution_payload.timestamp == compute_time_at_slot(state, state.slot)
+
+    assert validate_execution_payload(execution_payload)
```

Comment:
What I don't like about `is_` is that it usually a prefix of getters which are pure functions while this function is impure in case of stateful execution. We can use `verify_` if it's more suitable.

Reply:
You're right

`is_` is innappropriate for the mutation currently. *might* be reasonable if it were stateless but `verify_` works well